README.md

```diff
@@ -403,6 +404,88 @@ use Omnimail\Omnimail;
 $mailer = Omnimail::create(Omnimail::AMAZONSES, ['accessKey' => $accessKey, 'secretKey' => $secretKey]);
 ```
 
+<a name="mass-mailings"></a>
+## Mass Mailings
+
+The mass mailing component is for interacting with mass mailing providers. Currently the code focusses on data retrieval, but in future it should also define creating and sending mass mailings.
+
+There are 2 functions currently described for the Mass mailings interface `getMailings` and `getRecipients`.
+
+__getMailings__
+
+```
+    $mailer = Omnimail::create('Silverpop')->getMailings();
+    $mailer->setUserName($userName);
+    $mailer->setPassword($password);
+    $mailer->setStartTimeStamp(strtotime('7 days ago'));
+    $mailer->setEndTimeStamp(strtotime('now'));
+    // Instead of using set methods an array can be passed into create -ie 
+    // Omnimail::create('Silverpop', array('UserName' => $userName...);
```

Comment:
I definitely prefer this way, since the credentials can have totally different names for different platforms. E.g. the Amazon example above uses 'accessKey' and 'secretKey'.